wiki/Tournaments/TSC/1/en.md

```diff
@@ -0,0 +1,100 @@
+---
+tags:
+- Taiko Suiji Cup
+- Taiko Suiji
```

Comment:
Not sure how tags work, but these keywords are already in the article (the two shown here).